### ISSUE TITLE
Unknown commands, mathrm

### ISSUE THREAD

**Issue by gustaphe** [Suavesito-Olimpiada/LibTeXPrintf.jl#5]
Two issues baked in one here. 
One is that it does not know `\mathrm`.
The other, more important for making this package feel robust, is that the error reporting for unrecognized TeX commands is cryptic:
```
julia> texprintf(raw"\mathrm{a}")
ERROR: ArgumentError: Unknown command (1x)
Stacktrace:
 [1] texprintf(::Base.TTY, ::String)
   @ LibTeXPrintf ~/.julia/dev/LibTeXPrintf/src/LibTeXPrintf.jl:329
 [2] texprintf(::String)
   @ LibTeXPrintf ~/.julia/dev/LibTeXPrintf/src/LibTeXPrintf.jl:333
 [3] top-level scope
   @ REPL[21]:1
```

I would prefer if it said "Unknown TeX command: `\mathrm`".

**Comment by stevengj** [comment]
upstream issue?

**Comment by stevengj** [comment]
Looks like `mathrm` was already fixed upstream: https://github.com/bartp5/libtexprintf/commit/79dc4dda5c43b31aa3dd68ed685b7e97e380db9b … https://github.com/JuliaPackaging/Yggdrasil/pull/6778?

**Comment by Suavesito-Olimpiada** [comment]
I'll update to upstream, thanks for checking. :D

**Comment by Suavesito-Olimpiada** [comment]
Fixed with #9
